cattle/agent/handler.py

```diff
@@ -89,7 +89,7 @@ def _get_response_data(self, obj):
                 pass
 
             if method is not None:
-                return method(obj)
+                return {type: method(obj)}
```

Comment:
@ibuildthecloud Would update it anyway. In fact I just found all the mocked data structure are wrongly marked resourceType as instance/volume/image rather than instanceHostMap/volumeStoragePoolMap/imageStoragePoolMap. I would correct them as well.